code/build.R

```diff
@@ -356,6 +359,56 @@ foreach(region = region_names) %dopar% {
   }
 }
 
+# Combine all cohesive networks (CN) into a single file
+# Define the root directory of coherent networks
+output_folder = glue::glue("outputdata/{date_folder}/")
+
+# Prepare to store all GeoJSON data
+all_CN_geojson = list()
+
+# Loop through the subfolders and read GeoJSON files
+subfolders = list.dirs(output_folder, full.names = TRUE, recursive = FALSE)
+for (folder in subfolders) {
+  coherent_networks_path = file.path(folder, "coherent_networks")
+  geojson_files = list.files(coherent_networks_path, pattern = "\\.geojson$", full.names = TRUE)
+  
+  for (geojson_file in geojson_files) {
+    geojson_data = sf::st_read(geojson_file) |>
+      sf::st_transform(crs = 4326) 
+    all_CN_geojson[[length(all_CN_geojson) + 1]] = geojson_data
+  }
+}
```

Comment:
I would rewrite this with a `lapply()` or `map_dfr()` but this looks like it works.